### ISSUE TITLE
A small issue on model training and evaluation

### ISSUE THREAD

**Issue by Harry-zzh** [YonghengSun1997/MSCA-Net#4]
Thank you for your work. 

I am curious about a question. For ISIC2018 dataset, why not training the model on the whole training set and evaluating its performance on the test set directly?
